ember-app/app/mixins/cards/sortable.js

```diff
@@ -13,6 +13,8 @@ var SortableMixin = Ember.Mixin.create(CardMoveMixin, {
     var newSortable = _self.get('newSortable');
     var sortableFx = newSortable ? "superSortable" : "sortable";
     this.$(".cards")[sortableFx]({
+      containment: "document",
+      tolerance: "pointer",
```

Comment:
I actually disagree with pointer for the cards, I don't think it's better 

 > "intersect": The item overlaps the other item by at least 50%.

The behavior should be based on the bounding box of the card not the pointer of the mouse. 

I would agree that the autoscrolling could use a little bit of work to be "better" but that should be in the superSortable stuff, not the mouse tolerance